labgrid/remote/client.py

```diff
@@ -820,13 +820,15 @@ def main():
     subparser = subparsers.add_parser('io',
                                       help="change (or get) a digital IO status")
     subparser.add_argument('action', choices=['high', 'low', 'get'])
+    subparser.add_argument('-v', '--verbose', action='count', default=0)
```

Comment:
Without the StepReporter there is no feedback for the user, on the other hand the StepReporter can be quiet verbose. I think it is useful to debug why a state transition is not working, but if your strategy is working as expected, you hardly need the extra information from the StepReporter